frontend/test/metabase-smoketest/admin.cy.spec.js

```diff
@@ -11,371 +11,395 @@ const new_user = {
+      cy.findByLabelText("Name").type("Demo Dash");
+      cy.findByLabelText("Description").type("Many demos live here");
+      cy.findByText("Create").click();
+
+      cy.contains("People per");
+      cy.findByText("This dashboard is looking empty").should("not.exist");
+
+      cy.findByText("Save").click();
+    });
+
+    it("should add a simple JOINed question as admin", () => {
+      cy.findByText("Ask a question");
+
+      cy.findByText("Ask a question").click();
+      cy.findByText("Simple question").click();
+      cy.findByText("Sample Dataset").click();
+      cy.findByText("Orders").click();
 
-      // =================
-      // should see questions currently in the "Our Analytics" collection as user
-      // =================
+      // Join tables
+      cy.get(".Icon-notebook").click();
 
-      cy.findByText("Browse all items").click();
-      cy.contains("My personal collection");
+      cy.findByText("Data");
+      cy.findByText("Showing").should("not.exist");
 
-      // =================
-      // should see dashboard in the "Our Analytics" collection as user
-      // =================
+      cy.findByText("Join data").click();
+      cy.findByText("People").click();
+      cy.findByText("Visualize").click();
 
-      cy.findByText("Dashboards").click();
-      cy.contains("Demo Dash 2");
+      // Summarize by State
+      cy.findAllByText("Summarize")
+        .first()
+        .click();
+      cy.findByText("State").click();
+      cy.findByText("Done").click();
 
-      // =================
-      // should create my own question as user
-      // =================
+      cy.get(".Icon-pinmap");
+      cy.get(".Icon-table").should("not.exist");
 
+      // Save question (not to a dashboard)
+      cy.findByText("Save").click();
+      cy.findByLabelText("Name")
+        .clear()
+        .type("Order Totals by State");
+      cy.findAllByText("Save")
+        .last()
+        .click();
+      cy.findByText("Not now").click();
+    });
+
+    it("should add a question with a default line visualization as admin", () => {
       cy.findByText("Ask a question").click();
 
-      cy.contains("Native query");
+      cy.findByText("Native query");
 
+      cy.findByText("Ask a question").click();
       cy.findByText("Simple question").click();
       cy.findByText("Sample Dataset").click();
-      cy.findByText("Reviews").click();
+      cy.findByText("Orders").click();
 
-      cy.get(".Button")
-        .contains("Summarize")
+      cy.findByText("Product ID");
+      cy.findByText("Pick your data").should("not.exist");
+
+      // Summarize by date ordered
+      cy.findAllByText("Summarize")
+        .first()
         .click();
       cy.get(".scroll-y")
-        .contains("Rating", { timeout: 20000 })
+        .contains("Created At")
         .click();
       cy.findByText("Done").click();
 
-      cy.contains("Auto binned");
-      cy.get(".Icon-bar");
+      cy.get(".Icon-line");
```

Comment:
We should validate that `.Icon-line` has more than `n` number of elements. I forget the exact syntax, but it's something like `.its('length').should('be.gt', 2)`

Reply:
I found the correct syntax: `.its('length').should('be.gte', 2)` however, it results in the attached cypress error (cypress can see there are two elements and says it's looking for two or more, but it still fails).

So, I've used `cy.get(".Icon-line").should("have.length", 2);`